plugins/kubernetes/app/models/kubernetes/resource.rb

```diff
@@ -89,7 +89,7 @@ def revert(previous)
       def delete
         return true unless exist?
         request_delete
-        backoff_wait([0.0, 0.1, 0.2, 0.5, 1, 2, 4, 8, 16], "delete resource") do
+        backoff_wait([0.0, 0.1, 0.2, 0.5, 1, 2, 4, 8, 16, 32], "delete resource") do
```

Comment:
is this in seconds? (or another unit?)
curious if we have metrics around how long it takes to take down all pods from a deployment (although I'm sure it various a lot from deployment to deployment)

also `sum of waits should be ~30s which is the default delete timeout` comment is longer now, yes?

Reply:
Oh, I guess the code comment answered my question about time unit, haha!